### PR TITLE
Fix a windows doc issue

### PR DESCRIPTION
**Pull request by Evalle**
Signed-off-by: evalle shmarnev@gmail.com

### CODE REVIEW

**Comment by Evalle** [comment]
@thaJeztah Fixed, but I don't like how it's split the line. Can you suggest something?

File: docs/installation/windows.md [hunk 1]
```diff
@@ -346,7 +346,7 @@ uses. You can do this with
 [puttygen](http://www.chiark.greenend.org.uk/~sgtatham/putty/download.html):
 
 1. Open `puttygen.exe` and load ("File"->"Load" menu) the private key from
-  `%USERPROFILE%\.ssh\id_boot2docker`
+  `%USERPROFILE%\.docker\machine\machines\<name_of_your_machine>`
```
Comment (thaJeztah) [inline docs/installation/windows.md]: You could try changing to (make sure there are 8 spaces before `%USERPROFILE%`);

```Markdown
1. Open `puttygen.exe` and load ("File"->"Load" menu) the private key from this directory:

        %USERPROFILE%\.docker\machine\machines\<name_of_your_machine>

```

**Comment by Evalle** [comment]
@thaJeztah thank you for advice. Now it looks great. I hope you like it too :)

**Comment by thaJeztah** [comment]
Thanks @Evalle! I think it looks good.

A tip to check documentation changes; Have you checked if it looks good as well when the docs are generated? If you have access to docker on your computer, simply run `make docs` in the root of your git repository, then visit `http://<ip-address-of-your-docker-host>:8000` in your browser.

LGTM

ping @moxiegirl

**Comment by Evalle** [comment]
@thaJeztah I do these steps after every change of documentation file. Just to be sure ;)

**Comment by thaJeztah** [comment]
@Evalle o, great! Not all contributors are aware of that, so I thought I'd mention it, just in case :+1:

**Comment by SvenDowideit** [comment]
LGTM - though @Evalle if you can merge the 2 commits into one, that would be awesome :)

**Comment by Evalle** [comment]
@SvenDowideit here you go!

**Comment by thaJeztah** [comment]
Thanks @SvenDowideit, good catch, I missed that :-)

re-LGTM thanks @Evalle !

**Comment by thaJeztah** [comment]
ping @moxiegirl if you're going to push a docs update, could you include this PR?

**Pull request merged**